osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/application/threads/RecoverKeyProcessTest.java

```diff
@@ -85,14 +92,15 @@ public void testWhenDeviceNotFoundThenException() throws OsgpException {
   public void testWhenNoNewKeysThenNoActivate() throws OsgpException {
```

Comment:
Since there is no check in RecoverKeyProcess.run anymore if there are new keys, I would rename this test. Suggestion: `testWhenNotAbleToConnectWithNewKeys`